addons/kotsadm/1.10.2/install.sh

```diff
@@ -230,7 +230,8 @@ EOF
 
     openssl req -newkey rsa:2048 -nodes -keyout kotsadm.key -config kotsadm.cnf -x509 -days 365 -out kotsadm.crt -extensions v3_ext
 
-    kubectl -n default create secret generic kotsadm-tls --from-file=tls.key=kotsadm.key --from-file=tls.crt=kotsadm.crt --from-literal=acceptAnonymousUploads=1
+    kubectl -n default create secret tls kotsadm-tls --key=kotsadm.key --cert=kotsadm.crt
+    kubectl -n default annotate secret kotsadm-tls acceptAnonymousUploads=1
```

Comment:
I like the change, but I believe it should go as part of kotsadm 1.11.0, not 1.10.x
Also the latest in 1.10.x is 1.10.3, so this wouldn't be the patch to change in either case